scripts/process-reboot-cause

```diff
@@ -69,6 +70,45 @@ def read_reboot_cause_files_and_save_state_db():
                 x = TIME_SORTED_FULL_REBOOT_FILE_LIST[i]
                 os.remove(x)
 
+def read_dpu_reboot_cause_files_and_save_chassis_state_db():
+    # Get platform using device_info.get_platform()
+    platform_info = device_info.get_platform_info()
```

Comment:
The code to query DPUs from the platform.json should be part of `src/sonic-py-common/sonic_py_common/device_info.py`. There is an implementation for getting the [get_num_dpus](https://github.com/sonic-net/sonic-buildimage/blob/master/src/sonic-py-common/sonic_py_common/device_info.py#L895), so we can put something like `get_dpus_info`. This will prevent a code duplication in all places that need to access DPUs information

Reply:
@oleksandrivantsiv will do

Reply:
@oleksandrivantsiv done